node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java

```diff
@@ -150,7 +150,8 @@ private static Duration computeScalingDuration(Cluster cluster, ClusterSpec clus
         for (ScalingEvent event : cluster.scalingEvents()) {
             if (event.duration().isEmpty()) continue;
             completedEventCount++;
-            totalDuration = totalDuration.plus(event.duration().get());
+            // Assume we have missed timely recording completion if it is longer than 4 days
+            totalDuration = totalDuration.plus(maximum(Duration.ofDays(4), event.duration().get()));
```

Comment:
This should be minimum?

Reply:
The duration shall be maximum 4 days. Isn't that clear?

Reply:
Uhh, I guess. I thought this worked same as `Math.max()`, i.e. it returned the max of the 2 args, but this depends on the position of args and duration-minus-isNegative isn't the easiest thing to read.